Tella/Scenes/Settings/Views/Security/DeleteAfterFailView.swift

```diff
@@ -36,6 +36,14 @@ struct DeleteAfterFailView: View {
             }, cancelLabel: LocalizableSettings.settLockTimeoutCancelSheetAction.localized, saveAction: {
                 settingsViewModel.saveDeleteAfterFail()
                 sheetManager.hide()
+                
```

Comment:
I think it's better to move the code of `BottomButtonsView` into a separate variable that returns this view

Reply:
You are right! The view got a little bit more complex after adding the display toast logic. I isolated the BottomButtonsView and created a function for the toast to avoid the over complexity in one view